modules/main-rules/pages/roll-dice-and-add-combat-bonuses.adoc

```diff
@@ -1,6 +1,9 @@
-= CC3. Roll dice and add combat bonuses
+= CC3. Make the combat resolution roll
 
+.Combat resolution roll
 Both players: Roll a D6 and add your close combat bonuses to the result.
 
 * The player with the highest total score wins the combat.
-* If the total scores are the same then the outcome hangs in the balance! Place one Blast marker on every detachment in the combat and then *roll again*.
\ No newline at end of file
+* If the total scores are the same then the outcome hangs in the balance! Place one Blast marker on every detachment in the combat and then *roll again*.
```

Comment:
There's an implicit step here, given that the Blast markers in a draw can change the balance of the combat. I think that we should make it explicit:

```suggestion
* If the total scores are the same then the outcome hangs in the balance!
Place one Blast marker on every detachment in the combat, and then proceed again from xref:work-out-close-combat-bonuses.adoc[].
```

Reply:
Looks good to me.